pkg/util/trace/impl/motrace/report_statement.go

```diff
@@ -324,18 +325,23 @@ func mergeStats(e, n *StatementInfo) error {
 
 	// Parse the strings to integers and add the values together
 	for i := 1; i < len(eStatsElements); i++ {
-		eVal, err := strconv.Atoi(strings.TrimSpace(eStatsElements[i]))
+		eVal, err := strconv.ParseUint(strings.TrimSpace(eStatsElements[i]), 10, 0)
```

Comment:
If here you return a new moerr.NewInvalidInput(...), the error framewokr will log the error.

There is no need line 85-88.    Here the error is closer to where it happens and is will/can contain better message or information.

Reply:
got it. I will remove the error print in 85 - 88.